packages/destination-actions/src/destinations/moloco-rmp/common/fields.ts

```diff
@@ -0,0 +1,313 @@
+      required: false,
+    },
+    advertisingId: {
+      label: 'Advertising ID',
+      description: 'For app traffic, IDFA of iOS or ADID of android should be filled in this field. (e.g., 7acefbed-d1f6-4e4e-aa26-74e93dd017e4)',
+      type: 'string',
+      required: false,
+    },
+    uniqueDeviceId: {
+      label: 'Unique Device ID',
+      description: `For app traffic, a unique identifier for the device being used should be provided in this field.
+  Clients can issue identifiers for their user devices or use their IDFV values if using iOS apps.
+  The length of this id should not exceed 128 characters.`,
+      type: 'string',
+      required: false,
+    },
+    model: {
+      label: 'Model',
+      description: 'Device model, which is taken from the device without manipulation or normalization. (e.g., "iPhone 11 Pro")',
+      type: 'string',
+      required: false,
+    },
+    ua: {
+      label: 'User Agent',
+      description: 'User Agent. (e.g., "Mozilla/5.0 (iPhone; CPU iPhone OS 14_4_2 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Mobile/111FFF")',
+      type: 'string',
+      required: false,
+    },
+    language: {
+      label: 'Language',
+      description: 'ISO-639-1 alpha-2 language code. (e.g., "en")',
+      type: 'string',
+      required: false
+    },
+    ip: {
+      label: 'IP Address',
+      description: 'IP in IPv4 format. (e.g., 216.212.237.213)',
+      type: 'string',
+      required: false,
+    }
+  },
+  default: {
+    os: { '@path': '$.context.os.name' },
+    osVersion: { '@path': '$.context.os.version' },
+    advertisingId: { '@path': '$.context.device.advertisingId' },
+    uniqueDeviceId: { '@path': '$.context.device.id' },
+    model: { '@path': '$.context.device.model' },
+    ua: { '@path': '$.context.userAgent' },
+    ip: { '@path': '$.context.ip' }
+  }
+}
+
+export const SESSION_ID: InputField = {
+  label: 'Session ID',
+  description: 'Identifier for tracking users regardless of sign-in status. The length should not exceed 128 characters.',
+  type: 'string',
+  required: false,
+  default: {
+    '@path': '$.anonymousId'
+  }
+}
+
+export const DEFAULT_CURRENCY: InputField = {
+  label: 'Default Currency',
+  description: `The default currency value. If this is set, it will be used as a default currency value for items.
+  Available options are the followings
+  UNKNOWN_CURRENCY: Unknown currency.
+  USD: US Dollar.
+  KRW: Korean Won.
+  JPY: Japanese Yen.
+  EUR: EU Euro.
+  GBP: British Pound.
+  SEK: Swedish Krona.
+  INR: India Rupee.
+  THB: Thailand Baht.
+  IDR: Indonesia Rupiah.
+  CNY: China Yuan.
+  CAD: Canada Dollar.
+  RUB: Russia Ruble.
+  BRL: Brazil Real.
+  SGD: Singapore Dollar.
+  HKD: Hong Kong Dollar.
+  AUD: Autrailia Dollar.
+  PLN: Poland Zloty.
+  DKK: Denmark Krone.
+  VND: Viet Nam Dong.
+  MYR: Malaysia Ringgit.
+  PHP: Philippines Peso.
+  TRY: Turkey Lira.
+  VEF: Venezuela Bolívar.`,
+  type: 'string',
+  required: false
+}
```

Comment:
I recommend offering a list of choices, like this: 

```suggestion
export const DEFAULT_CURRENCY: InputField = {
  label: 'Default Currency',
  description: `The default currency value. If this is set, it will be used as a default currency value for items.',
  choices: [
  {label: 'UNKNOWN_CURRENCY', value: 'UNKNOWN_CURRENCY'},
  {label: 'USD', value: 'USD'},
  {label: 'KRW', value: 'KRW'},
  {label: 'JPY', value: 'JPY'},
  {label: 'EUR', value: 'EUR'},
  {label: 'GBP', value: 'GBP'},
  {label: 'SEK', value: 'SEK'},
  {label: 'INR', value: 'INR'},
  {label: 'THB', value: 'THB'},
  {label: 'IDR', value: 'IDR'},
  {label: 'CNY', value: 'CNY'},
  {label: 'CAD', value: 'CAD'},
  {label: 'RUB', value: 'RUB'},
  {label: 'BRL', value: 'BRL'},
  {label: 'SGD', value: 'SGD'},
  {label: 'HKD', value: 'HKD'},
  {label: 'AUD', value: 'AUD'},
  {label: 'PLN', value: 'PLN'},
  {label: 'DKK', value: 'DKK'},
  {label: 'VND', value: 'VND'},
  {label: 'MYR', value: 'MYR'},
  {label: 'PHP', value: 'PHP'},
  {label: 'TRY', value: 'TRY'},
  {label: 'VEF', value: 'VEF'}
],
  type: 'string',
  required: false
}
```

Reply:
Much readable. Thank you!